package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.3.1",
   "description": "Track the extent of digital political advertising, discover which parties are targeting you",
   "main": "index.js",
-  "apiUrl": "https://wtm-api.whotargets.me/v1/",
+  "apiUrl": "http://localhost:3000/dev/",
```

Comment:
I'm guessing you didn't mean to update this?

Reply:
Oh yeah, I missed that. The good old `git add .`